AMAZON-INSTRUCTIONS.md

```diff
@@ -0,0 +1,33 @@
+## Instructions
```

Comment:
should we link to the docs on revenuecat.com?

Reply:
💯